io.github.gssh-linux.gssh.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.gssh-linux.gssh
```

Comment:
```suggestion
app-id: io.github.gssh_linux.gssh
```

Reply:
source: https://docs.flatpak.org/en/latest/conventions.html#application-ids

Reply:
I can't do that, the name of the org is gssh-linux

Reply:
you didn't read the doc through. It says `-` is replaced by `_` except in the last component.

Sadly I need to fix the linter to check for this.